examples/svelte/src/Item.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+    import { createEventDispatcher, tick } from 'svelte';
+
+    export let item;
+
+    export let editing = false;
+    const dispatch = createEventDispatcher();
```

Comment:
`editing` shouldn't be a prop, per other comments, and convention dictates (or at least suggests) putting things like event dispatchers after props but before component state

```suggestion
    const dispatch = createEventDispatcher();

    let editing = false;
```

Reply:
moved dispatch to the top!